### PR TITLE
Change dynamic Notch MinHz Default

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: https://github.com/betaflight/betaflight/pull/11827

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: src/js/fc.js
Comment (Asizon) [inline src/js/fc.js:879]: Firmwsre side is taged for 4.3

**Review by Asizon — CHANGES_REQUESTED** [review]

**Review by Asizon — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
